tests/views_integration/Tribe/Events/Views/V2/Views/__snapshots__/Day_ViewTest__test_render_empty__1.php

```diff
@@ -156,6 +156,17 @@ class="tribe-events-c-view-selector__list-item-link"
 		<span class="tribe-events-c-view-selector__list-item-text">
 			Day		</span>
 	</a>
+</li>
+					<li class="tribe-events-c-view-selector__list-item tribe-events-c-view-selector__list-item--latest-past">
+	<a
+		href="http://test.tri.be/?post_type=tribe_events&#038;eventDisplay=latest-past&#038;tribe-bar-date=2019-01-01"
+		class="tribe-events-c-view-selector__list-item-link"
+		data-js="tribe-events-view-link"
+	>
+		<span class="tribe-events-c-view-selector__list-item-icon tribe-common-svgicon tribe-common-svgicon--latest-past"></span>
+		<span class="tribe-events-c-view-selector__list-item-text">
+			Latest-past		</span>
```

Comment:
This should not have a dash on it. Where are we setting this, can we make sure that it says Latest Past instead?